app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class SessionsController < Devise::SessionsController
+  respond_to :json
+
+  private
+
+  def respond_with(resource, _opts = {})
+    render json: resource
+  end
+
+  def respond_to_on_destroy
+    head :ok
```

Comment:
Sets header to 200 when we logout, is there anything else we might want to do as well?